src/VisualStudio/CSharp/Impl/CSharpVSResources.resx

```diff
@@ -559,4 +559,10 @@
   <data name="Perform_additional_code_cleanup_during_formatting" xml:space="preserve">
     <value>Perform additional code cleanup during formatting</value>
   </data>
+  <data name="CSharp_Coding_Conventions" xml:space="preserve">
+    <value>C# Coding Conventions</value>
```

Comment:
consider 'Code Style Rules' since we call it 'Code Style' everywhere else.   (don't feel strongly on tihs.  but it feels like we should try to be consistent here.

Reply:
The naming is used from the example .editorconfig on our docs landing page for dotnet editorconfig documentation: [https://docs.microsoft.com/en-us/visualstudio/ide/editorconfig-code-style-settings-reference?view=vs-2017#example-editorconfig-file](https://docs.microsoft.com/en-us/visualstudio/ide/editorconfig-code-style-settings-reference?view=vs-2017#example-editorconfig-file). The terminology was approved by Kasey and used by Allison, who originally implemented it, and we decided to keep both in sync.

---
In reply to: [219239324](https://github.com/dotnet/roslyn/pull/30043#discussion_r219239324) [](ancestors = 219239324)

Reply:
Works for me!